### DISCUSSION TITLE
popupopened event is not there in types [tabulator-tables]

### DISCUSSION THREAD

**Discussion by saivinoth** [DefinitelyTyped/DefinitelyTyped#62682] [Issues with a @types package]
popupopened event is not there in types

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
Which package are you referring to?

**Reply by saivinoth** [reply]
@types/tabulator-tables [tabulator-tables]

**Reply by MysteryBlokHed** [reply]
If you edit the discussion title to include [tabulator-tables] like the bot said, the package owners will be pinged

**Reply by saivinoth** [reply]
Done

**Reply by Jojoshua** [reply]
Just open a PR with the fix and we'll review it
